pkg/trace/api/container_linux.go

```diff
@@ -0,0 +1,60 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package api
+
+import (
+	"context"
+	"net"
+	"net/http"
+	"syscall"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/trace/log"
+	"github.com/DataDog/datadog-agent/pkg/util/containers/v2/metrics"
+)
+
+type ucredKey struct{}
+
+const cacheExpire = 5 * time.Minute
```

Comment:
Can you add a comment explaining why 5 minutes is the right amount of time? I wouldn't know by looking at this why 5 minutes is more appropriate than 30 seconds or 30 minutes.